utils/active_timer.js

```diff
@@ -2,7 +2,32 @@ const cron = require("node-cron");
 
 var TIMER_POOLS = {};
```

Comment:
I see this isn't something that you changed, but do we need consider storing these somewhere else other than in memory?
Will it cause problems if `process_manager` restarts and we lose the `TIMER_POOLS`

Reply:
Good Question.

When this service starts (in `app.js`) it calls the `initialize_timer` handler for each of the site's tenants.  This will load the active timers from the DB into working memory and use those for the timers.

So the timer information is persisted in the DB, and make active on startup.